### ISSUE TITLE
Sharing track doesn't work

### ISSUE THREAD

**Issue by MikeNGarrett** [dasevilla/rdio-party#121]
The that one:
![screen shot 2015-05-13 at 11 23 43 am](https://cloud.githubusercontent.com/assets/394888/7614092/9df55250-f962-11e4-8aa4-3eb09f4e3839.png)

**Comment by BCWilsonDotCom** [comment]
Yeah, those buttons down there need some love. None of them do anything except for the down vote right now.
